README.md

```diff
@@ -1,71 +1,139 @@
-# Kurtosis-Charon setup
+# Kurtosis-Charon
+
+## Overview
+
+This project leverages [kurtosis](https://docs.kurtosis.com) to run a local setup of ethereum and beacon chains with DVT.
+It does not connect to real beacon chains and instead it creates a preconfigured local environment, similar to [Ganache](https://archive.trufflesuite.com/ganache/).
+Once started, you will have several instances of execution clients, consensus clients, charon and validator clients running altogether.
+What's important to us: kurtosis takes care of configuring and activating validator keys, we only need to import the keys to run DVT.
+
+Kurtosis supports all of the existing vendors: lighthouse, nimbus, prysm, teku and lodestar for both BN and VC roles.
+We only need to specify the desired combination of BN and VC. In all cases it uses `geth` for EL.
+
 ## Pre-requisites
 
-Install kurtosis-cli by following the instructions [here](https://docs.kurtosis.com/install).
-Ensure you have docker installed on your machine.
+You need to have docker installed on your machine, because everything is running in docker.
 
-MAKE SURE `kurtosis-cli` IS THE LATEST VERSION.
+Install `kurtosis-cli` by following the instructions [here](https://docs.kurtosis.com/install).
+This project updates frequently, make sure to update it to the latest version before you start.
 
+On MacOS, simply use `brew`:
+    
 ```shell
+brew install kurtosis-cli
+```
 
-## TL;DR
+## Usage
+
+1. Start the docker.
+
+2. Build the charon image from your local charon repository: `docker build . -t obolnetwork/charon-local`.
+
+3. If you want to use the specific charon release from Docker Hub:
 
 ```shell
-make geth-lighthouse
-make charon
-make run-charon-lodestar
+docker image pull obolnetwork/charon:v1.1-dev
+docker tag obolnetwork/charon:v1.1-dev obolnetwork/charon-local:latest
 ```
+> The script in this project expects charon image to have `obolnetwork/charon-local:latest` tag.
 
-## Setup and Run
-
-To set up and run the project, follow these steps:
-Below example runs a geth-lighthouse-charon-lodestar setup.
-Follow the `Makefile` to run other combinations.
+4. Decide on the desired combination of BN and VC you need for testing. Start kurtosis with EL+CL stack by running:
 
-Step 1: Run the following command to run geth-lighthouse setup:
-    
 ```shell
+# Pick one of these commands
 make geth-lighthouse
+make geth-lodestar
+make geth-teku
+make geth-nimbus
+make geth-prysm
 ```
 
-Step 2: Verify that the setup is running in your docker-desktop and wait for 10 seconds. Create a Charon setup by running the following command:
-    
+> Note: this command often fails if you have leftovers from the previous run. In that case, run `make clean` and try again.
```

Comment:
Might also be worth saying that sometimes docker network-related errors can only be fixed with a complete docker daemon restart.

Reply:
Added.